model/itemTab.cc

```diff
@@ -41,7 +41,7 @@ namespace minsky
                                          if (itemSelector(*i)) 
                                            {		                                 
                                              itemVector.emplace_back(*i);
-                                             if (auto p=(*i)->plotWidgetCast()) itemCoords.emplace(make_pair(*i,make_pair(p->x(),p->y()))); 
+                                             if ((*i)->classType()=="PlotWidget") itemCoords.emplace(make_pair(*i,make_pair((*i)->x(),(*i)->y()))); 
```

Comment:
plotWidgetCast() was better. String comparisons are more expensive.

Reply:
IIRC, itemSelector is virtual, and returns true only if i points to the right item type. So the type test on this line and next is redundant. Just insert the coordinates into the itemCoords vector for all such tabbed items, even though we're not going to use them in the variable case.